auth-web/src/views/auth/HomeView.vue

```diff
@@ -16,30 +16,43 @@
 
       <header class="hero-banner">
         <v-container>
-          <h1>Welcome to Cooperatives Online<sup>Beta</sup></h1>
-          <p>File your BC cooperative association's annual reports and maintain your registered office addresses and director information.</p>
-
+          <h1>Start A Business And <br> Keep Records up to Date</h1>
+          <p>The Business Registry manages the creation (incorporation and registration)<br> and listing of businesses
+            and organizations in British Columbia.</p>
           <div class="hero-banner__cta-btns">
             <!-- Authenticated -->
-            <div v-if="userProfile">
-              <v-btn large color="#fcba19" class="cta-btn"
+            <div v-if="userProfile" class="cta-btns-authenticated">
+              <v-btn large color="#003366" class="cta-btn white--text"
+                href="https://www.bcregistrynames.gov.bc.ca/nro/" target="_blank" rel="noopener noreferrer">
+                Request a Name
+              </v-btn>
+              <v-btn large color="#003366" class="cta-btn white--text"
                 @click="goToManageBusinesses()">
-                Manage Businesses
+                Incorporate a Named Company
               </v-btn>
-              <v-btn large outlined color="#ffffff"
```

Comment:
Not super clear from this..i could be wrong...looks like the code for createAccount  is removed.. is createAccount button going away?

Reply:
There is now a link on opening page instead of a button, and according to the designs there is no longer a create account button once the user is in and authenticated. 

Here is a snap before the user logs in..

<img width="1425" alt="Screen Shot 2020-05-15 at 8 37 05 AM" src="https://user-images.githubusercontent.com/53542131/82068771-4cb67a00-9687-11ea-9bf9-dbcfc7678044.png">

Reply:
Here is a snap once their logged in..
Do you think the Create account is still required once a user is logged in?
I mentioned this to Scott and it sounded like it was handled in another way..?

<img width="1427" alt="Screen Shot 2020-05-15 at 8 38 15 AM" src="https://user-images.githubusercontent.com/53542131/82068896-753e7400-9687-11ea-97c7-02b54a51d168.png">

Reply:
hm..I am not sure..may @jeznorth or @sumesh-aot  will have input on this..but there should be a way to create account as well...we can just wait for their reviews

Reply:
K sounds good, i 100% agree, i feel like there should still be a way as well. Standing by on this one.

Reply:
Hey - you might want to consider removing the watermark (FPO)